modules/orchestrator/src/main/java/org/eclipse/xpanse/orchestrator/OrchestratorService.java

```diff
@@ -84,17 +85,17 @@ public void onApplicationEvent(ApplicationEvent event) {
      * @param oclLocation the location of the OCL descriptor.
      * @throws Exception if registration fails.
      */
-    public void registerManagedService(String oclLocation) throws Exception {
+    public void registerManagedServiceDeprecated(String oclLocation) throws Exception {
```

Comment:
instead of renaming the method.. can we simply delete it? since it is anyways not used by any real users.